spec.emu

```diff
@@ -53,9 +53,12 @@ contributors: "Jack Works"
             1. If _end_ is not *+∞* or *-∞* and if Type(_end_) is not _type_, throw a *TypeError* exception.
             1. If _start_ is *+∞* or *-∞*, throws a *RangeError* exception.
             1. Let _ifIncrease_ be `end > start`
+            1. Let _inclusiveEnd_ be *false*
```

Comment:
Is it on purpose that all items are numbered 1?

Reply:
The spec render will rewrite it in the rendered spec so I can easily reorder the steps